src/scripts/main.js

```diff
@@ -1,7 +1,267 @@
+/* eslint-disable no-shadow */
 'use strict';
 
 // Uncomment the next lines to use your game instance in the browser
 // const Game = require('../modules/Game.class');
 // const game = new Game();
 
-// Write your code here
+class Game {
```

Comment:
why you wrote class Game in main.js?
move code to /modules/Game.class and import here (uncomment code above)